### DISCUSSION TITLE
can someone pls show example on how to use the auth feature of rueidis?

### DISCUSSION THREAD

**Discussion by ouvaa** [redis/rueidis#497] [Q&A]
can someone pls show example on how to use the auth feature of rueidis?

**Comment by rueian** [comment]
Hi @ouvaa, what auth feature do you want to use?

**Reply by ouvaa** [reply]
@rueian sorry for late reply. basically just to auth command against redis server.

**Reply by rueian** [reply]
https://github.com/redis/rueidis/blob/7f213103b2ea88d429125a851a875a39d288fa5a/rueidis.go#L87-L88

You just need to fill the username and password here.
